_posts/en/newsletters/2023-10-04-newsletter.md

```diff
@@ -0,0 +1,170 @@
+    example too much of the funds in the channel between Alice and Bob
+    now belongs to Alice---the imbalance could be resolved by performing
+    an offchain [peerswap][] in the state contract.  E.g., Carol could
+    provide some funds to Alice in the state contract contingent on
+    Alice forwarding the same amount of funds through Bob to Carol in
+    the ordinary LN channel---restoring balance to the LN channel
+    between Alice and Bob.
+
+    One advantage of this approach is that nobody needs to know about the
+    state contract except the participants in each particular contract.
+    To all ordinary LN users, and all forwarding nodes not involved in a
+    particular contract, LN continues to operate using the current
+    protocol.  Another advantage, this time compared to existing channel
+    rebalancing operations, is that the state contract approach allows a
+    large number of forwarding nodes to maintain a direct peer
+    relationship for a small amount of onchain space, likely eliminating
+    any offchain rebalancing fees between those peers.  Keeping
+    rebalancing fees minimal makes it much easier for forwarding nodes
+    to keep their channels balanced, which improves their revenue
+    potential and makes sending payments across LN more reliable.
+
+    A downside to the approach is that it requires a multiparty state
+    contract, which is something that has never been implemented in
+    production before (to the best of our knowledge).  ZmnSCPxj mentions
+    two contract protocols that might be useful to use as a basis,
+    [LN-Symmetry][topic eltoo] and [duplex payment channels][].
+    LN-Symmetry would require a consensus change, which seems unlikely
+    to happen in the near future, so a [follow-up post][zmnscpxj
+    sidepools2] by ZmnSCPxj appears to be focusing on duplex payment
+    channels (which ZmnSCPxj calls "Decker-Wattenhofer" after the
+    researchers who first proposed them).  A downside of duplex payment
+    channels is that they can't be kept open indefinitely, although
+    ZmnSCPxj's analysis indicates they can probably be kept open for
+    long enough, and through enough state changes, to amortize their
+    cost effectively.
+
+    There were no public replies to the posts at the time of writing,
+    although we learned from private correspondence with ZmnSCPxj that
+    he is working on further developing the idea.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.17.0-beta.rc6][] is a release candidate for the next major
+  version of this popular LN node implementation.  A major new
+  experimental feature planned for this release, which could likely
+  benefit from testing, is support for "simple taproot channels".
+
+<!-- FIXME:harding to update Tuesday -->
```

Comment:
FYI, If their blog post is useful:
https://lightning.engineering/posts/2023-10-03-lnd-0.17-launch/